query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/TableDefinition.java

```diff
@@ -118,13 +118,13 @@ public static TableDefinition parse(
       PostgresColumnSpec spec;
       // todo: replace this with call to attribute service
       if (mapFields.contains(physName)) {
-        spec = new PostgresColumnSpec(physName, ValueType.STRING_MAP);
+        spec = new PostgresColumnSpec(physName, ValueType.STRING_MAP, false);
       } else if (bytesFields.contains(physName)) {
-        spec = new PostgresColumnSpec(physName, ValueType.BYTES);
+        spec = new PostgresColumnSpec(physName, ValueType.BYTES, false);
       } else if (tdigestFields.contains(physName)) {
-        spec = new PostgresColumnSpec(physName, ValueType.TDIGEST);
+        spec = new PostgresColumnSpec(physName, null, true);
```

Comment:
The default type is a string. so here as well, it will be `ValueType.String`.

Reply:
Done